homeworks/Vitalii.Lynnyk_VitaliiLynnyk/homework_3/function.js

```diff
@@ -0,0 +1,16 @@
+function start(){
+    let leftSide = "";
+    let rightSide = "";
+    let height = parseInt(prompt("Write a height, please = "),10);
```

Comment:
Please, don't forget for now we use node.js. "prompt" is not exist in node.js environment if you want to organize input I would recommend fetching input from command line argument.

Reply:
ok, i'm sorry, i will  change the code tomorrow.I want to asked, is it necessary to initialize variable, or this is a mistake?